packages/plugin-backgroundcolor/test/BackgroundColor.test.ts

```diff
@@ -166,53 +166,25 @@ describePlugin(BackgroundColor, testEditor => {
                     contentAfter: '<p>a[bc]d</p>',
                 });
             });
-            it('should set the background color of two characters to white, within a paragraph with yellow background', async () => {
-                await testEditor(BasicEditor, {
-                    contentBefore: '<p style="background-color: yellow;">a[bc]d</p>',
-                    stepFunction: async editor => {
-                        await uncolorBackground(editor);
-                    },
-                    contentAfter:
-                        '<p style="background-color: yellow;">a[<span style="background-color: white;">bc]</span>d</p>',
-                });
-            });
```

Comment:
Why did you choose to remove this test instead of skipping it and adding a comment, like for the others?
If I understand this right, this behavior doesn't exist anymore because we now only treat the background-color change at the character level, so that uncolorBackground would rightly do nothing because the _characters_ have no background color. Is that correct?

Reply:
I think I removed them because the behaviour was not relevant anymore, as opposed to a choice in spec that could be made in the future.
On top of that I also felt like uncolouring should remove the colour that was set on the element, as opposed to colouring it white. As in, if I have a red background on the paragraph, I highlight some text in green, then uncolour that text, it's pretty surprising to find it highlighted in white instead.

Your assessment seems correct.

Reply:
:+1: